src/platform_impl/linux/window.rs

```diff
@@ -215,7 +215,12 @@ impl Window {
     window.set_visible(attributes.visible);
     window.set_decorated(attributes.decorations);
 
-    window.set_keep_above(attributes.always_on_top);
+    if attributes.always_below_bottom && !attributes.always_on_top {
+      window.set_keep_below(attributes.always_below_bottom);
+    }
+    if attributes.always_on_top && !attributes.always_below_bottom {
+      window.set_keep_above(attributes.always_on_top);
+    }
```

Comment:
I think we should return an Err if both are specified
```rs
if attributes.always_on_top && attributes.always_below_bottom {
	return Err("Always on top and always on bottom, are both specified, you should only set one.");
}
```
Either that, or in `WindowBuilder` we can set the other one false depending on which one was called last
```rs
pub fn with_always_on_top(mut self, always_on_top: bool) -> Self {
    self.window.always_on_top = always_on_top;
    self.window.always_below_bottom = false;
    self
}
pub fn with_always_below_bottom(mut self, always_below_bottom: bool) -> Self {
    self.window.always_below_bottom = always_below_bottom;
    self.window.always_on_top = false;
    self
}
```

cc @wusyong

Reply:
lgtm. Just tested on macOS too and it works fine.

Reply:
3f4feba

Reply:
I cannot choose which error should be returned when "always on top" and "always on bottom" are both set, so I print warning instead.